libraries/app/application.cpp

```diff
@@ -1044,6 +1077,28 @@ void application::set_program_options(boost::program_options::options_descriptio
           "For database_api_impl::get_limit_orders to set its default limit value as 300")
          ("api-limit-get-order-book",boost::program_options::value<uint64_t>()->default_value(50),
           "For database_api_impl::get_order_book to set its default limit value as 50")
+         ("api-limit-lookup-accounts",boost::program_options::value<uint64_t>()->default_value(1000),
+          "For database_api_impl::lookup_accounts to set its default limit values as 1000")
```

Comment:
Actually the option doesn't set the default limit value, it sets the maximum.
Also, please don't include the default value in the message. It will be included automatically in `--help`, e. g.
```
> graphene/programs/witness_node/witness_node --help
Graphene Witness Node:
  -h [ --help ]                         Print this help message and exit.
...
  --api-limit-get-account-history-operations arg (=100)
                                        For history_api::get_account_history_op
                                        erations to set its default limit value
                                        as 100
```
Please fix the existing messages as well.

Reply:
Made the update